tests/compose/test-installroot.sh

```diff
@@ -12,6 +12,11 @@ treefile_set "boot-location" '"new"'
 # to be able to interact with the compose output directly, feed it back to
 # rpm-ostree, etc... So we just run whole scripts inside the VM.
 
+case $(stat -f -c '%T' .) in
+    fuseblk) fatal "This test will not work on a fuse filesystem: $(pwd)" ;;
```

Comment:
Ahh yup, I have this in my bash history:

> TMPDIR=/srv/tmpdir TESTS=basic-unified V=1 ./tests/compose.sh                                                                                                                                                                                                           

Should we mention `TMPDIR` in this error message?

Reply:
I'll roll that into a followup PR.

Reply:
On second thought...the fix I used is just `sudo mount --bind /run/host/var/tmp /var/tmp/` which is actually what coretoolbox does today.  So I think this is a `toolbox` bug - admittedly one that would manifest when using `podman` directly too.  Basically there should be a mode for `podman` that sets up e.g. `/tmp` and `/var/tmp` to be host mounts in a dynamic subdir, much like how `systemd-run -p PrivateTmp` works.  IMO the solution isn't to override `TMPDIR` everywhere because that's a losing battle, so we shouldn't reference that in the error.